packages/react-pose/src/_tests/index.test.tsx

```diff
@@ -118,6 +118,43 @@ test('posed: `onPoseComplete` gets called with pose as argument', () => {
   });
 });
 
+test('posed: `onPoseComplete` gets called for children', () => {
+  return new Promise(resolve => {
+    const Group = ({ pose = 'exit' }) => (
+      <PoseGroup>
+        <Parent key='a' pose={pose}>
+          <Child
+            onPoseComplete={pose => {
+              expect(pose).toBe('enter');
+              resolve();
+            }}
+          />
+        </Parent>
+      </PoseGroup>
+    )
+    const wrapper = mount(<Group />)
+    wrapper.setProps({ pose: 'enter' })
+  })
+});
+
+test('posed: `onPoseComplete` gets called for mounting children', () => {
```

Comment:
would be good to add tests for regular posed elements, not only for PoseGroup children

i.e. for such tree
```
<Sidebar initialPose="closed" pose="opened">
  <Content onPoseComplete={() => {}}>
  </Content>
</Sidebar>
```